### ISSUE TITLE
ReferenceError: module is not defined

### ISSUE THREAD

**Issue by yfaway** [mzagorny/gas-local#8]
Hello,

I am having this strange issue. I am new to nodejs as well so it could be entirely an environment issue. Here is the code (the file contains exactly these two lines):

var gas = require('gas-local');
var glib = gas.require('../');

When running with mocha, I got this error: 
module.exports = asString
^
ReferenceError: module is not defined
    at ..//xxx/node_modules/date-format/lib/index.js:3:1
    at ContextifyScript.Script.runInContext (vm.js:35:29)
    at Object.runInContext (vm.js:89:6)
    at Object.gasrequire [as require] (C:\git\brain-dump\node_modules\gas-local\gasrequire.js:42:12)
    at Object.<anonymous> (C:\git\brain-dump\test\DataManagerTest.js:3:16)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)

I would appreciate if you could give me some pointer on why this is happening. I sync with GAS using the npm gas module.

Thanks,

**Comment by mzagorny** [comment]
Hi @yfaway Sorry for very long answer! Have you figured out what was the reason?

**Comment by yfaway** [comment]
Hello @mzagorny ,

Unfortunately, as you mentioned, it's been so long ago. I can't remember anything. Most likely I moved on with other options (i.e. it wasn't solved). Here's something I documented; I am not quite sure if it would help you:

Error: Cannot find module 'chai'
Caused: either package wasn’t installed or was installed globally. Need to run this:
npm install --save-dev chai
The -g or --global option is really for independent program running on terminal (i.e. traditional executable program).
https://stackoverflow.com/questions/36838834/node-js-cannot-find-module-chai

Sorry that's all I could provide. I haven't touched nodejs for almost a year now.
